Sources/Public/Animation/LottieView.swift

```diff
@@ -162,6 +161,94 @@ public struct LottieView: UIViewConfiguringSwiftUIView {
     }
   }
 
+  /// Returns a copy of this view updated to display the given `AnimationProgressTime`.
+  ///  - If the `currentProgress` value is provided, the `currentProgress` of the
+  ///    underlying `LottieAnimationView` is updated. This will pause any existing animations.
+  ///  - If the `animationProgress` is `nil`, no changes will be made and any existing animations
+  ///    will continue playing uninterrupted.
+  public func currentProgress(_ currentProgress: AnimationProgressTime?) -> Self {
+    configure { view in
+      if
+        let currentProgress = currentProgress,
+        view.currentProgress != currentProgress
+      {
+        view.currentProgress = currentProgress
+      }
+    }
+  }
+
+  /// Returns a copy of this view updated to display the given `AnimationFrameTime`.
+  ///  - If the `currentFrame` value is provided, the `currentFrame` of the
+  ///    underlying `LottieAnimationView` is updated. This will pause any existing animations.
+  ///  - If the `currentFrame` is `nil`, no changes will be made and any existing animations
+  ///    will continue playing uninterrupted.
+  public func currentFrame(_ currentFrame: AnimationFrameTime?) -> Self {
+    configure { view in
+      if
+        let currentFrame = currentFrame,
+        view.currentFrame != currentFrame
+      {
+        view.currentFrame = currentFrame
+      }
+    }
+  }
+
+  /// Returns a copy of this view updated to display the given time value.
+  ///  - If the `currentTime` value is provided, the `currentTime` of the
+  ///    underlying `LottieAnimationView` is updated. This will pause any existing animations.
+  ///  - If the `currentTime` is `nil`, no changes will be made and any existing animations
+  ///    will continue playing uninterrupted.
+  public func currentTime(_ currentTime: TimeInterval?) -> Self {
+    configure { view in
+      if
+        let currentTime = currentTime,
+        view.currentTime != currentTime
+      {
+        view.currentTime = currentTime
+      }
+    }
+  }
+
+  /// Returns a view that updates the given binding each frame with the animation's `realtimeAnimationProgress`.
+  /// The `LottieView` is wrapped in a `TimelineView` with the `.animation` schedule.
+  ///  - This is a one-way binding. Its value is updated but never read.
+  ///  - If provided, the binding will be updated each frame with the `realtimeAnimationProgress`
+  ///    of the underlying `LottieAnimationView`. This is potentially expensive since it triggers
```

Comment:
Hmm, in the future we might want the ability to specify a level of granularity for how often you receive updates. Say every second or half second.

Reply:
That sounds like it could be useful. We can definitely add it later if a use case comes up.